File: Palindrome/Palindrome.py
def is_palindrome(s: str) -> bool:
    s = s.lower().replace(" ", "")  # normalize string
    return s == s[::-1]

# Example usage
print(is_palindrome("madam"))      # True
print(is_palindrome("racecar"))    # True
print(is_palindrome("hello"))      # False

File: Palindrome/Palindrome.js
function isPalindrome(str) {
    str = str.toLowerCase().replace(/\s+/g, "");
    return str === str.split("").reverse().join("");
}

// Example usage
console.log(isPalindrome("madam"));    // true
console.log(isPalindrome("racecar"));  // true
console.log(isPalindrome("hello"));    // false
